### DISCUSSION TITLE
Attempt to use supabase locally fails

### DISCUSSION THREAD

**Discussion by camilosw** [supabase/supabase#29371] [Questions]
I'm following this guide https://supabase.com/docs/guides/cli/getting-started, but after running `supabase init` and `supabase start` it failed. I used `--debug` with the last command and this is the part of the log that I think is relevant:

DEBUG Import 'https://deno.land/x/jose@v4.13.1/index.ts' failed: error sending request for url (https://deno.land/x/jose@v4.13.1/index.ts). Retrying...
DEBUG Import 'https://deno.land/std/http/status.ts' failed: error sending request for url (https://deno.land/std/http/status.ts). Retrying...
DEBUG starting new connection: https://deno.land/
DEBUG starting new connection: https://deno.land/
failed to create the graph
Error: main worker boot error: worker boot error: failed to create the graph

CLI version: 1.192.5
Docker version: 4.34.2 (I configured it according to the previous guide)
macOS: 13.6.7

Is anyone else experiencing the same issue? How can I make it work?

**Comment by tazmaniax** [comment]
Yes, I'm getting this error as well, but only after I made a change to my Edge function code. However, no attempt at isolating the offending part has helped and the function serve debug log isn't very helpful. The change involved introducing Drizzle ORM so included some npm imports. Reverting back to the working code and adding those npm imports doesn't cause the issue so it's something else or more complicated. There were quite a few code changes so reapplying the changes will be extremely time consuming. This has completely blocked me.

CLI version: 1.200.3
OrbStack: 1.7.3
OS: macOS 15.1

```
% supabase functions serve --env-file ./supabase/.env.local        
Setting up Edge Functions runtime...
Serving functions on http://127.0.0.1:54321/functions/v1/<function-name>
Using supabase-edge-runtime-1.58.3 (compatible with Deno v1.45.2)
serving the request with /Users/xxx/git/yyy/zzz/supabase/functions/webhook
failed to create the graph
An error has occured
InvalidWorkerCreation: worker boot error: failed to create the graph
    at async UserWorker.create (ext:sb_user_workers/user_workers.js:145:15)
    at async Object.handler (file:///root/index.ts:154:22)
    at async respond (ext:sb_core_main_js/js/http.js:163:14) {
  name: "InvalidWorkerCreation"
}
```

**Reply by tazmaniax** [reply]
Ok, I just managed to fix it by prefixing the npm imports with "npm:". I thought if the dependency was specified in the import_map.json with the "npm:" prefix then it was not needed when referencing the dependency in the source code, but it would appear that it is required in the source code import reference. I hope that helps.
```
{
  "imports": {
    "drizzle-orm": "npm:drizzle-orm@0.33.0",
    "postgres": "npm:postgres@3.4.4",
  }
}
```

```
import postgres from "npm:postgres";
import { drizzle, PostgresJsDatabase } from "npm:drizzle-orm/postgres-js";
```

**Reply by AntonOfTheWoods** [reply]
Pretty sure that means it is completely disregarding the map file.

**Comment by nela** [comment]
@camilosw Did you figure it out? stuck with the same error?
